template_helper.go

```diff
@@ -56,6 +59,36 @@ func jsonPath(expr string, tmpl string) (ret string, err error) {
 	return "", nil
 }
 
+func gJsonPath(expr string, tmpl string) (ret string, err error) {
+	defer func() {
+		if r := recover(); r != nil {
+			err = r.(error)
+		}
+		logrus.WithFields(logrus.Fields{
+			"err":  err,
+			"tmpl": tmpl,
+			"expr": expr,
+		}).Debug("running gJsonPath")
+	}()
+
+	if tmpl == "" {
+		return "", nil
+	}
+
+	if !gjson.Valid(tmpl) {
+		return "", errors.New("Invalid json")
+	}
+
+	node := gjson.Parse(tmpl).Get(expr)
+	if node.Exists() {
+		if node.Type.String() == "String" {
+			return node.String(), nil
+		}
```

Comment:
is this string type check necessary? If the type is "String", will `Raw` also return just the string content? Otherwise, we may also need to consider other types, e.g. bool, number, map and etc.

Reply:
`node.Raw` holds an escaped string for simple values which is why the special handling of strings.
If we return `node.Raw` for [this test case](https://github.com/checkr/openmock/pull/81/files#diff-993d8b65865a4d3a304c60b5093e863aR63), this happens
```--- FAIL: TestGJsonPath (0.00s)
    template_helper_test.go:63:
        	Error Trace:	template_helper_test.go:63
        	Error:      	Not equal:
        	            	expected: "John"
        	            	actual  : "\"John\""

        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -1 +1 @@
        	            	-John
        	            	+"John"
        	Test:       	TestGJsonPath
```

It doesn't affect the template tests though.